src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonPropertiesView.ts

```diff
@@ -134,7 +167,57 @@ export class ExecutionPlanComparisonPropertiesView extends ExecutionPlanProperti
 		if (this._model.secondaryElement?.properties) {
 			secondaryProps = this._model.secondaryElement.properties;
 		}
-		this.populateTable(columns, this.convertPropertiesToTableRows(primaryProps, secondaryProps));
+
+		const tableRows = this.convertPropertiesToTableRows(primaryProps, secondaryProps);
+		this.setSummaryElement(this.getExpensivePropertySummary(tableRows));
+		this.populateTable(columns, tableRows);
+	}
+
+	private getExpensivePropertySummary(tableRows: { [key: string]: string }[]): string[] {
+		let summary: string[] = [];
+
+		tableRows.forEach(row => {
+			const rowName = row.name['text'];
+			if (row.primary && row.secondary) {
+				const primaryText = row.primary['text'].split(' ');
+				const secondaryTitle = row.secondary['title'].split(' ');
+
+				if (primaryText.length === secondaryTitle.length && primaryText.length <= 2 && secondaryTitle.length <= 2) {
+					const MAX_PROPERTY_SUMMARY_LENGTH = 3;
+
+					for (let i = 0; i < primaryText.length && summary.length < MAX_PROPERTY_SUMMARY_LENGTH; ++i) {
+						if (this.isNumber(primaryText[i]) && this.isNumber(secondaryTitle[i])) {
+							const primaryValue = Number(primaryText);
+							const secondaryValue = Number(secondaryTitle);
+
+							let summaryPoint: string;
+							if (primaryValue > secondaryValue) {
+								summaryPoint = this._orientation === ExecutionPlanCompareOrientation.Horizontal
+									? getTopPlanIsGreaterThanBottomPlanSummaryPointTemplate(rowName)
+									: getLeftPlanIsGreaterThanRightPlanSummaryPointTemplate(rowName);
+							}
+							else {
+								summaryPoint = this._orientation === ExecutionPlanCompareOrientation.Horizontal
+									? getBottomPlanIsGreaterThanTopPlanSummaryPointTemplate(rowName)
+									: getRightPlanIsGreaterThanLeftPlanSummaryPointTemplate(rowName);
+							}
+
+							summary.push(summaryPoint);
+						}
+					}
+				}
+			}
+		});
+
+		return summary;
+	}
+
+	private isNumber(text: string): boolean {
+		if (typeof text !== 'string') {
```

Comment:
Given that we say the type is a string this check isn't needed (type checking should mostly be preventing this).

So I'd either remove this check or make the paramter parameter unknown 

`export function isNumber(obj: unknown): boolean {`